src/pluto/pluto.py

```diff
@@ -421,111 +321,319 @@ def generate_graph(v1, v2):
-    else:
-        st.markdown(data["version_text"])
+        applied_corrections = data['pluto_corrections_applied']
+        not_applied_corrections = data['pluto_corrections_not_applied']
 
-    # EXPECTED VALUE
-    st.header("Expected Value Comparison")
-    st.write(
-        "if nothing showed up, then it means there aren't any expected value change"
-    )
+        if applied_corrections is None or not_applied_corrections is None:
+            st.info("There are no available corrections reports for this branch.")
+            return
+            
+        st.markdown(
+            """
+            PLUTO is created using the best available data from a number of city agencies. To further
+            improve data quality, the Department of City Planning (DCP) applies changes to selected field
+            values.
+
+            Each Field Correction is labeled for a version of PLUTO. For programmatic changes, this is version in which the programmatic change was
+            implemented. For research and user reported changes, this is the version in which the BBL
+            change was added to PLUTO_input_research.csv.
+
+            For more information about the structure of the pluto corrections report,
+            see the [Pluto Changelog Readme](https://www1.nyc.gov/assets/planning/download/pdf/data-maps/open-data/pluto_change_file_readme.pdf?r=22v1).
+            """
+        )
+
+        version_dropdown = np.insert(np.flip(np.sort(data["pluto_corrections"].version.dropna().unique())), 0, 'All')
+        version = st.sidebar.selectbox("Filter the field corrections by the PLUTO Version in which they were first introduced", version_dropdown)
+        
+        st.subheader("Manual Corrections Applied", anchor="corrections-applied")
+        
+        create_corrections_section(applied_corrections, version, applied=True)
 
-    create_expected(data["df_expected"], v1, v2)
+        st.subheader("Manual Corrections Not Applied", anchor="corrections-not-applied")
+
+        st.markdown(
+            """ 
+            For each record in the PLUTO Corrections table, PLUTO attempts to correct a record by matching on the BBL and the 
+            Old Value column. As the underlying datasources change and improve, PLUTO records may no longer match the old value 
+            specified in the pluto corrections table. The graph and table below outline the records in the pluto corrections table that failed to be applied for this reason.
+            """
+        )
+        create_corrections_section(not_applied_corrections, version, applied=False)
+
+        st.info(
+            """
+            See [here](https://www1.nyc.gov/site/planning/data-maps/open-data/dwn-pluto-mappluto.page) for a full accounting of the changes made for the latest version
+            in the PLUTO change file.
+            """
+        )
+
+    report_type = st.sidebar.selectbox("Choose a Report Type", ["Compare with Previous Version", "Review Manual Corrections"])
```

Comment:
This is the logic for the different pages of PLUTO - all existing pluto logic (unchanged) is in the version_comparison_report function below.

Reply:
ok word, thanks for helping me navigate this. My first comment before I dig into the new content is that this should be moved up to live below the `get_data` call. My view of how `pluto.py` should be structured:
 1. interactive elements that let user chose what content they want to see
 2. function definitions
 3. calls to those function 

Does this seem like a good plan to you?

Reply:
Yeah that seems reasonable!